### PR TITLE
Changes corresponding to new swagger PR for API:2018-01-10

### PR DESCRIPTION
**Pull request by avneeshrai**
Updating sdk-for-net corresponding to swagger PR:
https://github.com/Azure/azure-rest-api-specs/pull/2419

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-		<Version>1.0.3-preview</Version>
+		<Version>2.0.0-preview</Version>
```
Comment (shahabhijeet) [inline src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj]: Add package release notes

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Missing meta data files
1) Use generate.cmd to generate your SDK
2) Once you generate using the latest stable version of Autorest (generates .txt file that is needed)
3) Build your generated SDK on command line to generate additional meta data file (*.props that is needed)
msbuild build.proj /t:build /p:Scope=SDKs\RecoveryServices.SiteRecovery

**Comment by avneeshrai** [comment]
@shahabhijeet Would update with the changes as soon as we have swagger PR merged

**Comment by avneeshrai** [comment]
@shahabhijeet I have updated the PR taking your comments (Swagger PR https://github.com/Azure/azure-rest-api-specs/pull/2419 is also merged ). However, I observed issues with generate.cmd as mentioned by Vipin in the attached mail so used the usual way to generate sdk. And have updated the .props file manually (it was not getting updated by running msbuild).

File: src/SDKs/RecoveryServices.SiteRecovery/AzSdk.RP.props [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>RecoveryServices_2016-08-10;</AzureApiTag>
+    <AzureApiTag>RecoveryServices_2018-01-10;RecoveryServices_2016-08-10;</AzureApiTag>
```
Comment (shahabhijeet) [inline src/SDKs/RecoveryServices.SiteRecovery/AzSdk.RP.props]: @avneeshrai this should only have 1 version.
So basically it should only reflect 2018 API version, assuming that is the newest version.

File: src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj (hunk 1 above)
Comment (shahabhijeet) [reply src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj]: @avneeshrai if you are in preview, you don't have to bump up major version.
Your new version can be 1.1.0-preview

File: src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Recovery Services SiteRecovery Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Recovery Services SiteRecovery Resources.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.3.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyFileVersion("2.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Properties/AssemblyInfo.cs]: @avneeshrai and only assemblyFileVersion changes to 1.1.0

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/RecoveryServices.SiteRecovery/AzSdk.RP.props (hunk 2 above)
Comment (avneeshrai) [reply src/SDKs/RecoveryServices.SiteRecovery/AzSdk.RP.props]: Done

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**